src/lua/minifio.lua

```diff
@@ -0,0 +1,113 @@
+local ffi = require('ffi')
+local minifio = require('internal.minifio')
+
+-- Several file manipulation functions without dependencies on
+-- tarantool's built-in modules to use at early initialization
+-- stage.
+
+ffi.cdef([[
+    char *
```

Comment:
Minor:
As for me, the following format is more readable:
```c
char *dirname(char *path);
```
Feel free to ignore.

Reply:
I think I should follow our [C Style Guide](https://www.tarantool.io/en/doc/latest/dev_guide/c_style_guide/#chapter-6-functions):

> Note that we place the function return type on the line before the name and signature.